API/Api/Api.ts

```diff
@@ -36,14 +43,13 @@ class WhiteToothAPI {
 
     parseRequestURL(url: string): ParsedRequest {
         const urlParts = url.split("/");
-        if (urlParts.length !== 3)
```

Comment:
from one side - consider set this to some const with a meaningful name
from the other side - maybe this should be obvious..
so - your choise